test/es6/function.name.js

```diff
@@ -761,6 +761,17 @@ var tests = [
             assert.areEqual("get", desc.get.name);
             assert.areEqual("set", desc.set.name);
         }
+    },
+	{
+        name: "Function name will be set only when LHS is identifier reference.",
+        body: function()
+        {
+            var obj = {};
+            obj.foo = function() {
+                assert.areEqual(obj.foo.name, "");

```

Comment:
what happens when we have something like 
```js
var a = function b() {}. 
```
What does `a.name` return? Can we have a test case for that as well as for `function b() {}; b.name` case?